### PR TITLE
Wait (for await)

### PR DESCRIPTION
**Pull request by amccool**
should need an await before accessing the result

### CODE REVIEW

File: src/Documentation/Advanced-Concepts/Powershell-Client.md [hunk 1]
```diff
@@ -146,7 +150,7 @@ PS> Stop-GrainClient
 
 ```
 
-Thats it for now. We plan to update this page as we introduce more Cmdlets like use Observers, Streams and other Orleans core features more natively on Powershell.
+That is it for now. We plan to update this page as we introduce more Cmdlets like use Observers, Streams and other Orleans core features more natively on Powershell.
```
Comment (ReubenBond) [inline src/Documentation/Advanced-Concepts/Powershell-Client.md]: Could we also change the last sentence to: We plan to update this page as we introduce Cmdlets for other Orleans features such as Observers and Streams. ?

**Comment by ReubenBond** [comment]
Thanks, @amccool :) Could you also remove the blank lines at the top of each powershell snippet?
image

It's rendering poorly on DocFx

File: src/Documentation/Advanced-Concepts/Powershell-Client.md (hunk 1 above)
Comment (amccool) [inline src/Documentation/Advanced-Concepts/Powershell-Client.md]: so just removing the `Thats it for now` blurb?

Comment (ReubenBond) [inline src/Documentation/Advanced-Concepts/Powershell-Client.md]: @amccool no, that part is fine - I just meant to replace the sentence after it

File: src/Documentation/Advanced-Concepts/Powershell-Client.md [hunk 2]
```diff
@@ -137,7 +133,11 @@ PS> $grainId = [System.Guid]::Parse("A4CF7B5D-9606-446D-ACE9-C900AC6BA3AD")
 
 PS> $grain = Get-Grain -GrainType $grainInterfaceType -GuidKey $grainId
 
-PS> $message = $grain.SayHelloTo("Gutemberg").Result
+PS> $sayHelloTask = $grain.SayHelloTo("Gutemberg")
+
+PS> $sayHelloTask.Wait();
+
+PS> $message = $sayHelloTask.Result
```
Comment (jdom) [inline src/Documentation/Advanced-Concepts/Powershell-Client.md]: not sure I understand this change. Unless Wait() behaves differently in Powershell, it is doing a blocking call until the result is calculated, and then you just access the pre-calculated Result. Why not just call Result, which will block until the result is calculated too? Seems easier to have 1 line than 3.
Of course, if you want to parallelize, then that's another scenario, and you might want to store each task separately, but not necessarily use Wait() either, just Task.WaitAll.

Comment (amccool) [inline src/Documentation/Advanced-Concepts/Powershell-Client.md]: I will try to double check.  BTW it looks like psgallery does not have the module published

**Pull request merged**